contents/newsletter/remote-working.md

```diff
@@ -0,0 +1,99 @@
+
+How do you build a culture of trust? Here’s a checklist.
+
+* **Give people autonomy.** At PostHog, we hire people who work well with high-level direction, and who are comfortable stepping on toes to get things done. We trust people to do their best work and, in return, they deliver.
+
+* **Honest feedback.** GitLab has a [policy](https://handbook.gitlab.com/handbook/values/) of “blameless problem solving.” Simply put, when something goes wrong, the investigation of the root cause is, as the company puts it, blameless. This encourages everyone to speak up without fear of punishment. People are trusted to make mistakes and, as a result, problems that do arise are easier to solve.
+
+* **Work in public.** At Zapier, trust and transparency are a big part of the [company’s values](https://zapier.com/jobs/culture-and-values-at-zapier). As co-founder and CEO Wade Foster explains: “Transparency promotes equity and inclusion within our team and customer base. So work in public. Have discussions in the open and document your work.” Successful remote companies build everything out in the open. This gives everyone the context they need, and eliminates the political squabbles that plague less transparent companies.
+
+* **Shared values.** The most successful remote teams have a shared set of values and behaviors – and these are often clearly defined in a public handbook. These values and behaviors are more important than dictatorial processes. As Buffer co-founder and CEO Joel Gascoigne wrote [almost a decade ago](https://buffer.com/resources/why-transparency/), “transparency breeds trust, and trust is is the foundation of great teamwork.” It’s still true today.
+
+### 4. They make meeting up easy
+
+Pretty much all successful remote teams have one thing in common: they’re not remote all the time.
+
+That doesn’t mean hybrid working, or mandatory office days, it means both encouraging meeting up and making it easy to do so.
+
+At **PostHog**, we achieve this in a bunch of different ways:
+
+* **$300 monthly co-working budget**.** This makes it easy for people to meet up and work together if they already live in the same city.
+
+* **$1,500 quarterly meet-up budget** for ad hoc in-person collaboration with colleagues. This can cover flights, accommodation, and other expenses. At PostHog, Paul and Ben used this recently to meet for a week in Italy and ship our new [heatmap and scroll depth feature](https://posthog.com/changelog/2024#mouse-and-scroll-depth-tracking-launched).
```

Comment:
```suggestion
* **$1,500 quarterly meet-up budget** for ad hoc in-person collaboration with colleagues. This can cover flights, accommodation, and other expenses. At PostHog, Paul and Ben used this recently to meet for a week in Italy and ship our new [heatmap and scroll depth feature](/changelog/2024#mouse-and-scroll-depth-tracking-launched).
```